### PR TITLE
Lint fixes #1 torchao/dtypes

### PR DESCRIPTION
**Pull request by jainapurva**
Performed lint fixes in directory: torchao/dtypes

- Test: 
   - ruff check .
   - ruff format --check

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]

File: torchao/dtypes/fpx/fpx.py [hunk 1]
```diff
@@ -36,8 +51,40 @@ def _bit_interleave(x: Tensor, n_bits: int, undo: bool = False) -> Tensor:
 
     if not undo:
         bit_order = {
-            1: [1, 5, 9, 13, 17, 21, 25, 29, 3, 7, 11, 15, 19, 23, 27, 31,
-                0, 4, 8, 12, 16, 20, 24, 28, 2, 6, 10, 14, 18, 22, 26, 30],
+            1: [
+                1,
```
Comment (janeyx99) [inline torchao/dtypes/fpx/fpx.py:55]: lol this is a fun result of this lint...

Comment (jainapurva) [reply torchao/dtypes/fpx/fpx.py:55]: This is auto-formatted in ruff. If I change the format, the CI/CD test for ruff format --check fails.

File: torchao/dtypes/affine_quantized_tensor.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import torch
 from typing import Tuple, Optional, Union
-from collections import defaultdict
-import functools
+import torchao.ops
```
Comment (msaroufim) [inline torchao/dtypes/affine_quantized_tensor.py:3]: I suspect this line broke our binary builds

Comment (jerryzh168) [reply torchao/dtypes/affine_quantized_tensor.py:3]: yeah is this caught by normal PR CI btw?

File: torchao/dtypes/fpx/fpx.py (hunk 1 above)
Comment (gau-nernst) [reply torchao/dtypes/fpx/fpx.py:55]: does ruff support some kind of skipping a block of code? I know black has `# fmt: off/on`. This is one of the cases where auto-format tools make things worse 😄

**Pull request merged**